autoload.php

```diff
@@ -1,10 +1,23 @@
 <?php
 
-// For older (pre-2.7.2) verions of google/apiclient
 if (
+    class_exists('Google\\Client')
+    || (
+        class_exists('Composer\\InstalledVersions')
+        && version_compare(\Composer\InstalledVersions::getVersion('google/apiclient'), '2.7.2', '>')
+    )
+) {
+    return;
```

Comment:
Isn't this saying that if the version is greater than 2.7.2, we don't want to load the legacy namespaces?

That's not the case - we _do_ want to load the legacy namespaces even in >2.7.2, otherwise we would break existing implementations.

Reply:
You are correct, that logic wasn't correct and actually I'm gonna remove that condition. It will be probably better also not to introduce additional dependency with the `getVersion()` method.